### PR TITLE
Update publication_quality_figures.rst

### PR DESCRIPTION
**Pull request by abhisht51**
Made the tone more formal

### CODE REVIEW

**Comment by abhisht51** [comment]
For #45

File: docs/source/tutorials/publication_quality_figures.rst [hunk 1]
```diff
@@ -44,33 +44,25 @@ which completely resides on Python tools.
...
-   scripting (I am sure you love it as much as I do). It may take some
-   more time at the beginning, but with
-   the advantage that you will not have to repeat the process when,
-   for some reason, you need to modify the plots you generated
+   The basic operations are similar to what one would do in a vector
+   graphics editor but using scripts instead of a mouse cursor. This reduces repitition as one will not have to repeat the process when,
+   for some reason, one needs to modify the plots they generated
    with matplotlib (to add more data or modify the
-   parameters of your analysis, just to name a few reasons).
+   parameters to the analysis, for example).
```
Comment (btel) [inline docs/source/tutorials/publication_quality_figures.rst:65]: i think "to the analysis" makes little sense, what is meant here is that we might want to modify the parameters that we used to generate the figures

File: docs/source/tutorials/publication_quality_figures.rst [hunk 2]
```diff
@@ -44,33 +44,25 @@ which completely resides on Python tools.
 
       ``anscombe.py``
 
-2. *Export to SVG*
+2. *Exporting to SVG*
 
-   A nice feature of matplotlib is that it allows to export figure to
+   A nice feature of matplotlib is that on can export figures to
```
Comment (btel) [inline docs/source/tutorials/publication_quality_figures.rst:49]:
```suggestion
   A nice feature of matplotlib is that one can export figures to
```

**Review by btel — CHANGES_REQUESTED** [review]
thanks for your contriution. It looks very good. I just made a few comments. We can merge once you fix them. Thanks again!

**Comment by btel** [comment]
sure, if I can just ask you to wait a few days, because I will use it to teach some people about pull request, etc. this week, so I guess it would be better to have some work left for them ;-) if you want please have a shot at other issues as well !

**Comment by btel** [comment]
@abhisht51  I also did not see any changes for the other comments, so I unresolved them

File: docs/source/tutorials/publication_quality_figures.rst (hunk 1 above)
Comment (abhisht51) [reply docs/source/tutorials/publication_quality_figures.rst:65]: Would "parameters of the existing analysis" make sense?

**Comment by btel** [comment]
perfect, thanks!

**Pull request merged**